legal-api/src/legal_api/models/filing.py

```diff
@@ -127,12 +127,41 @@ class Source(Enum):
     }
 
     __tablename__ = 'filings'
+    # this mapper is used so that new and old versions of the service can be run simultaneously,
+    # making rolling upgrades easier
+    __mapper_args__ = {
```

Comment:
Where is it used?

Reply:
it's used by SQLAlchemy as part of the framework.
i'll expand on the comments here.